src/components/organize/people/PersonDetailsCard.tsx

```diff
@@ -10,9 +10,7 @@ import { LocationCity, Mail, Phone } from '@material-ui/icons';
 import PersonCard from './PersonCard';
 import { ZetkinPerson } from 'types/zetkin';
 
-const PersonDetailsCard: React.FunctionComponent<{ person: ZetkinPerson }> = ({
-  person,
-}) => {
+const PersonDetailsCard: React.FunctionComponent<ZetkinPerson> = (person) => {
```

Comment:
I was stumped by this interface. More details in comment where this is actually used.